### PR TITLE
Offshoot : Collection browser layout broken for rendertron (?noscript=true) 

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-5266

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "description": "The Internet Archive Collection Browser.",
   "license": "AGPL-3.0-only",
   "author": "Internet Archive",
-  "version": "0.2.0",
+  "version": "0.2.2",
```
Comment (nsharma123) [inline package.json:6]: tag 0.2.1 was already created by clau so decided to go with 2.2

File: package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "@internetarchive/feature-feedback": "^0.1.2",
     "@internetarchive/field-parsers": "^0.1.3",
     "@internetarchive/histogram-date-range": "^0.1.6",
-    "@internetarchive/infinite-scroller": "^0.1.2",
+    "@internetarchive/infinite-scroller": "0.1.3",
```
Comment (mc2) [inline package.json:28]: Suggest "^0.1.3"

About using the caret ^: https://nodesource.com/blog/semver-tilde-and-caret/

**Review by mc2 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (dualcnhq) [reply package.json:28]: @nsharma123 is there a specific reason why you removed the caret here?

i also just learned this versioning, more info on source [here](https://javascript.plainenglish.io/the-math-behind-npm-versioning-5236f8c821c2)

image

**Comment by dualcnhq** [comment]
would recommend to create a separate PR commit for the version bump to 0.2.2 so we have clear release tag changelog

**Review by dualcnhq — APPROVED** [review]

**Pull request merged**